source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/OverflowActionLayoutRenderer.java

```diff
@@ -66,17 +66,22 @@ public View renderActions(final RenderedAdaptiveCard renderedCard, final Context
         final IOverflowActionRenderer overflowActionRenderer = CardRendererRegistration.getInstance().getOverflowActionRenderer();
         final List<View> menuItemList = renderSecondaryActionElements(baseActionElementList, renderedCard, context, fragmentManager, viewGroup, cardActionHandler, hostConfig, renderArgs);
 
-        View overflowActionView;
-        if (overflowActionRenderer == null || (overflowActionView = overflowActionRenderer.onRenderOverflowAction(viewGroup, menuItemList, renderArgs.isRootLevelActions())) == null)
+        View overflowActionView = null;
+        try
         {
-            overflowActionView = renderDefaultOverflowAction(context, viewGroup, hostConfig);
+            if (overflowActionRenderer == null || (overflowActionView = overflowActionRenderer.onRenderOverflowAction(viewGroup, menuItemList, renderArgs.isRootLevelActions())) == null) {
+                overflowActionView = renderDefaultOverflowAction(context, viewGroup, hostConfig);
+            }
+            final LinearLayout contentLayout = new LinearLayout(context);
+            contentLayout.setLayoutParams(new LinearLayout.LayoutParams(ViewGroup.LayoutParams.MATCH_PARENT, ViewGroup.LayoutParams.WRAP_CONTENT));
+            contentLayout.setOrientation(LinearLayout.VERTICAL);
+            setDrawables(contentLayout);
+            overflowActionView.setOnClickListener(new OverflowActionOnClickListener(contentLayout, overflowActionRenderer, menuItemList));
+        }
+        catch (Exception e)
```

Comment:
Is there a specific exception we are expecting, or is this just to continue rendering?

Reply:
This is for to continue rendering. Since client also does the custom overflow rendering, we want to make sure  card rendering should not be broken incase of exception.